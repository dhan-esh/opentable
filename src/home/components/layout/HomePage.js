import React, { Component } from "react";
import HomeContainer from "../container/HomeContainer";
import SearchBar from "../container/SearchBar";


class HomePage extends Component {
    state = {  }
    render() {
        return (
            <React.Fragment>
                <SearchBar />
                <HomeContainer/>
            </React.Fragment>
        );
    }
}

export default HomePage;