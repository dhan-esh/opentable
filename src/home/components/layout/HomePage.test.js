import React from "react";
import { shallow } from "enzyme";
import HomePage from "./HomePage";
import "../../../setupTests";

describe("HomePage Component", () => {
  it("renders HomePage", () => {
    const wrapper = shallow(<HomePage />);
    expect(wrapper).toMatchSnapshot();
  });
});

