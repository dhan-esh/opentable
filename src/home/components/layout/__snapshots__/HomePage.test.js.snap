// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage Component renders HomePage 1`] = `
<Fragment>
  <Connect(SearchBar) />
  <Connect(opentableApi) />
</Fragment>
`;
